src/reducers/projects.js

```diff
@@ -25,13 +25,14 @@ const newProject = Immutable.fromJS({
     javascript: '',
   },
   enabledLibraries: new Immutable.Set(),
+  minimizedComponents: new Immutable.Set(),
```

Comment:
Hah, nice! &#8275;synergy&#8275;

Do you think I should change the vocabulary for the actions, too? I.e.,

* `MINIMIZE_COMPONENT` &rarr; `HIDE_COMPONENT`
* `minimizeComponent()` &rarr; `hideComponent()`